### DISCUSSION TITLE
Ugly gap appears when viewing PR from mobile

### DISCUSSION THREAD

**Discussion by adamlui** [community/community#80524] [Pull Requests]
### Select Topic Area

Bug

### Body

In Android Chrome, sometimes this unzoomable view occurs

![Untitled](https://github.com/community/community/assets/10906554/ea4590bd-ecfb-472e-9558-b21bd752d979)

The phone is a 2023 Motorola G Stylus

**Comment by Develops01** [comment]
simply use github app

**Reply by adamlui** [reply]
I tried it once but uninstalled it, the UI is not very coordinated with desktop/mobile site

**Reply by adamlui** [reply]
Also that does not eliminate this bug's existence

**Reply by Develops01** [reply]
i dont have this bug not in the browser nor in the app , if there is problem it is all about your phone

**Reply by adamlui** [reply]
Nope it is not "all about your phone" it can potentially exist on any number of phones (just because your one phone can't replicate it does not suddenly mean other phones can't, you are not those other people you cannot speak for them)

**Reply by adamlui** [reply]
Also this bug not existing in your mobile app has nothing to do with anything (the report pertains to Android Chrome which is a web browser)

**Reply by adamlui** [reply]
Also you already forgot your proposed solution is use mobile app, implying it's not bugged already (so is redundant to repeat, in addition to being irrelevant due to report's explicitly stated platform)
